manuscript/book.md

```diff
@@ -39,21 +39,21 @@ for (let i = 0; i < names.length; i++) {
 }
 ```
 
-And now with the `.map()` method:
+And with the `.map()` method instead of a `for` loop:
 
 ```js
 const names = ['Bilbo Baggins', 'Gandalf', 'Gollum'];
 const kebabNames = names.map(name => _.kebabCase(name));
 ```
 
-We can shorten it even more if our processing function accepts only one argument, and [kebabCase from Lodash](https://lodash.com/docs#kebabCase) does:
+We can shorten it even more if our processing function accepts the value as first argument, [kebabCase from Lodash](https://lodash.com/docs#kebabCase) does:
 
 ```js
 const names = ['Bilbo Baggins', 'Gandalf', 'Gollum'];
 const kebabNames = names.map(_.kebabCase);
 ```
```

Comment:
````suggestion
```

This wouldn't work with functions that accept more than one argument, because the `.map()` also passes and array index as the second argument and a whole array as the third, like `parseInt()` that accepts the radix as the second argument:

```js
const inputs = ['1', '2', '3'];
inputs.map(parseInt); // -> [1, NaN, NaN]
inputs.map(value => parseInt(value)); // -> [1, 2, 3]
```

Here in the first example `.map()` calls `parseInt()` with an array index as a radix, which gives a wrong result. In the second example, we're explicitly passing only the value to the `parseInt()`, so it uses the default radix of 10.
````